pubsub/gcppubsub/example_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcppubsub_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"gocloud.dev/gcp"
+	"gocloud.dev/pubsub"
+	"gocloud.dev/pubsub/gcppubsub"
+	"golang.org/x/oauth2/google"
+)
+
+func Example() {
+	ctx := context.Background()
+	scope := "https://www.googleapis.com/auth/cloud-platform"
```

Comment:
Can you add some comments to all of this setup stuff? I suspect it's the most useful part of the example.